src/server/active_tran_server.cpp

```diff
@@ -247,6 +247,22 @@ active_tran_server::connect_to_page_server (const cubcomm::node &node, const cha
       return ER_NET_PAGESERVER_CONNECTION;
     }
 
+  auto ps_conn_error_lambda = [&node] ()
```

Comment:
Move `ps_conn_error_lambda` definition to the top of the function and use in the two above error cases too.

Reply:
done